src/util/BaseLoginController.js

```diff
@@ -21,6 +21,7 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
   }
 
   return Okta.Controller.extend({
+    className: 'base-login-controller',
```

Comment:
Hmm, I'm not sure this will work as we want it - i.e. it will be overwritten by any controller that extends it. What's the purpose of this?